firebase_admin/_user_mgt.py

```diff
@@ -280,6 +301,33 @@ def password_salt(self):
         return self._data.get('salt')
 
 
+class GetUsersResult:
+    """Represents the result of the ``auth.get_users()`` API."""
+
+    def __init__(self, users, not_found):
+        """Constructs a GetUsersResult.
+
+        Args:
+            users: List of `UserRecord` instances.
+            not_found: List of `UserIdentifier` instances.
+        """
+        self._users = users
+        self._not_found = not_found
+
+    @property
+    def users(self):
+        """Set of UserRecords, corresponding to the set of users that were
```

Comment:
Should this be backticked ``UserRecord`` objects?

Reply:
Done (here and immediately below).